### PR TITLE
avatar-component added

### PR DESCRIPTION
**Pull request by rishabh486**
pr for avatar component

### CODE REVIEW

File: avatar.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./avatar.css"/>
+</head>
+<body>
+    <div class="avatar-component">
+        <div class="badge-parent">
+            <div class="avatar-parent">
+            
+              <img  class="avatar-img" src="./daniil-onischenko-uFSZldf9xeM-unsplash.jpg"/>
```
Comment (priyam1103) [inline avatar.html]: Please add images, It is not visible. Applicable for all src's.

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    
+    width:10rem;;
+    height:10rem;
+    border-radius:50%;
+    overflow:hidden;
+    
+}
+.avatar-parent{
+    
+    width:12rem;;
```
Comment (priyam1103) [inline avatar.css]: width:12rem;

**Review by priyam1103 — APPROVED** [review]
@rishabh486 Things looks good. Please make indentation proper in html. It will make the code more readable.
And in CSS please remove unwanted spacing in files.

Also please add screenshots for UI components in PR.

**Comment by rishabh486** [comment]
Screenshot (27)

File: avatar.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  width: 100%;
+  object-fit: cover;
+  height: 100%;
+}
+.avatar img {
+  width: 100%;
+  object-fit: cover;
+  height: 100%;
+}
+/* .avatar-proj{
```
Comment (developer-anand) [inline avatar.css]: remove commented code.

File: avatar.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    flex-direction:column;
+} */
+.icon-button {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 50px;
+  height: 50px;
+  color: #333333;
```
Comment (developer-anand) [inline avatar.css]: use var for colors.

File: avatar.css [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  position: relative;
+  width: 5rem;
+  height: 5rem;
+}
+
+.avatar-badge {
+  position: absolute;
+  top: 6px;
+  right: 0px;
+  width: 22px;
```
Comment (developer-anand) [inline avatar.css]: use rem for values more than 10.

**Review by developer-anand — APPROVED** [review]
Good to merge with changes.

File: avatar.css (hunk 5 above)
Comment (rishabh486) [reply avatar.css]: fixed the rem values

File: avatar.css (hunk 4 above)
Comment (rishabh486) [reply avatar.css]: changed with variables

File: avatar.css (hunk 3 above)
Comment (rishabh486) [reply avatar.css]: removed the commented code

**Pull request merged**